app/script/components/list/conversationListCell.js

```diff
@@ -0,0 +1,83 @@
+/*
+ * Wire
+ * Copyright (C) 2017 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+'use strict';
+
+window.z = window.z || {};
+window.z.components = z.components || {};
+
+z.components.ConversationListCell = class ConversationListCell {
+  constructor({conversation, is_selected = z.util.noop, click = z.util.noop}) {
+    this.on_in_viewport = this.on_in_viewport.bind(this);
+
+    this.conversation = conversation;
+    this.is_selected = is_selected;
+    this.on_click = click;
+
+    this.entered_viewport = ko.observable(false);
+    this.user = ko.pureComputed(() => this.conversation.participating_user_ets()[0]);
```

Comment:
Should we optimize this and initiate user/users computed only based on group type?